README.md

```diff
@@ -161,20 +161,25 @@ https://console.developers.google.com: click on "Select a project..."
 in the top navbar and then "Create a project."
 
 You will then need a name for your AppEngine project.  Let's suppose
-you call it `mycompany-snippets`.
+you call it `mycompany-snippets`. You must then set the project by 
+running `gcloud config set project mycompany-snippets` from the snippets
```

Comment:
In most cases, this will end up actually setting the project for the default Google Cloud configuration globally.  Unfortunately the way the Google Cloud SDK [currently handles configurations](https://cloud.google.com/sdk/gcloud/reference/topic/configurations) is a bit of a mess, and there is no way to tie a configuration to a local project except manually.

Because of this, I'd request we keep the kinda hacky way we were doing it before an enforcing the project name is passed as an env variable along with the make deploy invocation.  It looks like the new way to manually specify on command line is via the global gcloud flag `--project`.  (It's not super elegant, but it will prevent someone from inadvertently breaking their default config from a different google cloud project.)